bedrock/sitemaps/views.py

```diff
@@ -1 +1,33 @@
-# Create your views here.
+from django.views.generic import TemplateView
+
+from bedrock.sitemaps.models import SitemapURL
+
+
+class SitemapView(TemplateView):
+    content_type = 'text/xml'
+
+    def _get_locale(self):
+        if self.kwargs['is_none']:
+            locale = ''
```

Comment:
Hmm shouldn't this be `locale = None` to work with line 28?

If so, I wonder why yours tests pass?

Reply:
the naming is confusing here. In the DB I store the empty string for URLs that don't have a locale. The `is_none` kwarg comes from the URL because the file that is for URLs without a locale is `/sitemap_none.xml`. I set it to `None` when the view is for the sitemap index, which is when it's not for the none file, and it's not for one of the files for a locale.

Reply:
I've added some more comments to hopefully make things more clear

Reply:
Got it! Still confusing that locale can be empty string but I get it. Thanks for the comments

Reply:
Do you think it'd be better if it was some other specific value? I had it be "none" for a bit but thought that was potentially even more confusing. I could have it be `null` in the DB. Just not sure how best to handle the URLs that don't have a locale. The ones I mean are here: https://www.mozilla.org/sitemap_none.xml e.g. https://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul

Reply:
I've updated things to use a special string and constant to indicate a URL has no locale. I think this makes the code clearer.